pkgs/development/python-modules/torch/default.nix

```diff
@@ -160,6 +165,7 @@ in buildPythonPackage rec {
   ];
 
   postPatch = lib.optionalString rocmSupport ''
+    rm cmake/Modules/FindCUDAToolkit.cmake
```

Comment:
@GaetanLepage you moved it into `postPatch`, but you moved it into block which happens when `rocmSupport` is enabled.

Instead, consider the change I did here to properly guard it: https://github.com/ConnorBaker/nixpkgs/commit/6d8c6ac313ec6327a0477238703a0f160fcec445#diff-59c22b0fc67d897077e55030166ca816d19c80b7767b2ad486bc0aaa2a772115R193-R197

Reply:
Oh right !
Done